core/transport/src/main/java/alluxio/util/grpc/GrpcChannel.java

```diff
@@ -14,9 +14,12 @@
 import io.grpc.CallOptions;
 import io.grpc.Channel;
 import io.grpc.ClientCall;
+import io.grpc.ConnectivityState;
```

Comment:
@ggezer what's the plan for these wrapper classes going forward? it seems wasteful to wrap every method we need since its no longer required for dependency management

Reply:
In boot-up and security PR, these wrappers are used as authentication layer and I'm currently using them for resource control purposes due to gRCP issues we have. They are not simple wrappers anymore and using them will automatically provide authentication for data path.